bootstrap/kubeadm/config/default/kustomization.yaml

```diff
@@ -0,0 +1,74 @@
+# Adds namespace to all resources.
+namespace: kubeadm-system
```

Comment:
Should this perhaps be capi-kubeadm instead? Or should we also call out that it is a bootstrap provider somehow? I'm just worried that using `kubeadm-system` might either cause confusion with the upstream kubeadm project or could eventually conflict if they use it as well for some of the operator work they are doing...

Reply:
capi-kubeadm-system?